### ISSUE TITLE
Color/face mismatch in Skills window borders (and background)

### ISSUE THREAD

**Issue by janitor-raus** [TRI-HD/TRI-HD-Project#1]
So here's the first problem noticed:
![image](https://github.com/TRI-HD/TRI-HD-Project/assets/11655762/cdc8b2c9-5a78-4a50-97d4-7d8c5911801f)

You can notice the miscoloration and subtle mismatch of faces on the corner piece, as well as the left end of the S patterns along the bottom. Likely upscaling artifacts.

The second issue lies with the background color of this whole Skills panel. Here's how it looks with vanilla textures:
![image](https://github.com/TRI-HD/TRI-HD-Project/assets/11655762/c720205d-169c-403f-a1a9-153357d4b3fd)

And here it is again with the pack enabled. Notice how the background color behind the grid of lines has become much fainter, and it's actually **entirely disappeared** on the left column where the skill names (not the Active Skills one) is:
![image](https://github.com/TRI-HD/TRI-HD-Project/assets/11655762/6043afc7-d6ed-47fa-8950-e70bb5f3b862)

Lastly as a bonus, the default clock/health/stamina segment of the HUD (potentially others too) have a white shadow around it so that it stands out even in dark environments, something currently absent from this pack. It'd be nice to re-apply that for consistency :)

How it looks in the pack:
![image](https://github.com/TRI-HD/TRI-HD-Project/assets/11655762/1bc43794-2376-4c59-bc58-8b1d6e5d9540)

And the default, with the subtle white glow/outline to make the elements stand out in the dark:
![image](https://github.com/TRI-HD/TRI-HD-Project/assets/11655762/c8e82038-59a9-4d74-9ab4-783eded15755)

**Comment by ZAIBATSUU** [comment]
Yes, I am aware of those issues, they are not Ai upscale Artifacts, but
rather an issue in the connection of frames between texture charts, kinda
tricky it will be fixed in the next patch (no release-date yet)

**Comment by janitor-raus** [comment]
Ah, the minor shape continuity where the black outlines aren't pixel perfectly aligned is one thing but that's not what I was referring to as a potential cause of upscaling. I am not 100% sure of what is hand-drawn after all, though (for example) the Wii instructions screen is for sure an upscale.

Just to be clear on my side in case of a misunderstanding, here's another sample to better showcase the color seams across the texture repetition due to the hue not being homogeneous across the surface. I screwed with the red channel curve to amplify the differences to the naked eye:

![image](https://github.com/TRI-HD/TRI-HD-Project/assets/11655762/6095cc15-b499-4695-b495-94c3a2933f26)

I imagine you were aware of the missing background layer in the equipment screen so I just gotta wait for that (and maybe a white glow/outline for the HUD elements) in the next update, whenever that may be. Thank you for your work thus far! :D
